core/src/ast_builder/table.rs

```diff
@@ -1,4 +1,5 @@
 use super::{CreateTableNode, DeleteNode, DropTableNode, SelectNode, ShowColumnsNode, UpdateNode};
+use crate::ast_builder::alter_table::AlterTableNode;
```

Comment:
Combine it with the import statement above
In addition, you need to attach the cfg of the alter-table feature.